yarn-project/circuits.js/src/barretenberg/crypto/ecdsa/index.test.ts

```diff
@@ -0,0 +1,46 @@
+import { CircuitsWasm } from '../../../index.js';
+import { Ecdsa } from './index.js';
+import { TextEncoder } from 'util';
+
+describe('ecdsa', () => {
+  let ecdsa!: Ecdsa;
+
+  beforeAll(async () => {
+    const wasm = await CircuitsWasm.get();
+    ecdsa = new Ecdsa(wasm);
+  });
+
+  it.skip('should verify signature', () => {
```

Comment:
are these meant to be skipped?

Reply:
Yes, they are currently not working. The issue is being looked at but shouldn't hold up this PR.